teps/0007-conditions-beta.md

```diff
@@ -348,6 +353,110 @@ spec:
       - echo-file-does-not-exist
 ```
 
+### Status
+
+Add `Skipped Tasks` section to the `PipelineRunStatus` that contains a map with `PipelineTaskName` as key and resolved `WhenExpressions` as value. Thus, users can know why a particular `Task` was skipped. In addition, `TaskRuns` from guarded `Tasks` that execute because their `WhenExpressions` evaluate to true would have the resolved `WhenExpressions` included in the `TaskRun` status. 
+
```

Comment:
Hey @jerop, sorry for delay, this sounds good but we have it implemented as `minimal skipped` with [Skipped Tasks ](https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1beta1/pipelinerun_types.go#L342) as a list of `SkippedTask`, should we update this section? 🤔

Reply:
good catch @pritidesai, updated it 😊